ct_app/tests/test2_SUITE.erl

```diff
@@ -0,0 +1,63 @@
+-module(test2_SUITE).
```

Comment:
It would be good to have at least some comments about the suites. Names like `test_SUITE`, `test2_SUITE`, `test3_SUITE` are not helpful...